### PR TITLE
AP_Notify: add support for blinking 1 LED for notify

### PR DESCRIPTION
**Pull request by peterbarker**
... also a simulator for it

### CODE REVIEW

**Comment by peterbarker** [comment]
Tested on SDMODELH7V1.

File: libraries/SITL/SIM_GPIO_LED_1.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    };
+
+    SIM_LED_n<1> leds{"GPIO_LED_1", colours};
+
+    uint8_t LED_A_PIN;
+};
+
+} // namespace SITL
+
+#endif  // AP_SIM_GPIO_LED_2_ENABLED
```
Comment (andyp1per) [inline libraries/SITL/SIM_GPIO_LED_1.h:46]: LED_1

Comment (peterbarker) [reply libraries/SITL/SIM_GPIO_LED_1.h:46]: Fixed, thanks

**Comment by IamPete1** [comment]
I know there is at least one board that defines a second LED pin which is not really connected just so they get a blinky light.

**Comment by peterbarker** [comment]
> I know there is at least one board that defines a second LED pin which is not really connected just so they get a blinky light.

Ah, great, thanks for the confirmation.  I thought, "surely we must blink LEDs on these boards that don't define 2+ LEDs".  But I accepted the insanity and added support, half-thinking someone would point out some alternate pathway through which we blinked LEDs!

**Comment by peterbarker** [comment]
Incidentally, suggestions for what patterns to show when are welcome.  I juts put something together which looked like a good launching point.

**Review by tridge — APPROVED** [review]

**Pull request merged**